lib/multiplexer/multiplexer.go

```diff
@@ -686,13 +686,14 @@ func NewPROXYSigner(signingCert *x509.Certificate, jwtSigner JWTPROXYSigner) (*P
 // SignPROXYHeader creates a signed PROXY header with provided source and destination addresses
 func (p *PROXYSigner) SignPROXYHeader(source, destination net.Addr) ([]byte, error) {
 	header, err := signPROXYHeader(source, destination, p.clusterName, p.signingCertDER, p.jwtSigner)
+	logger := log.WithFields(log.Fields{
+		"src_addr":     fmt.Sprintf("%v", source),
+		"dst_addr":     fmt.Sprintf("%v", destination),
+		"cluster_name": p.clusterName})
 	if errors.Is(err, ErrBadIP) {
-		log.WithFields(log.Fields{
-			"src_addr":     fmt.Sprintf("%v", source),
-			"dst_addr":     fmt.Sprintf("%v", destination),
-			"cluster_name": p.clusterName}).Warn("Got bad IP while trying to sign PROXY header")
+		logger.Warn("Got bad IP while trying to sign PROXY header")
 		return nil, nil
```

Comment:
There was a few cases when we had situation that was not really relevant to supported parts of IP pining but failed IP propagation. So it was a bit dangerous place which could affect all users instead of just pinned ones. I think we probably can remove it now, since all cases should be theoretically handled, at least for the testplan duration 🤔